examples/ex_aip.py

```diff
@@ -7,19 +7,19 @@
 # infile = '/proj/ads/abstracts/data/AIP/AIP.test/RSI/v91/i5/054901_1/Markup/VOR_10.1063_5.0005676.xml'
 # infile = '/proj/ads/abstracts/data/AIP/JATS.0127/JCP/v154/i2/024904_1/Markup/VOR_10.1063_5.0033645.xml'
 # infile = '/proj/ads/abstracts/data/AIP/JATS.0127/AJP/v89/i2/210_1/Markup/VOR_10.1119_10.0002365.xml'
-infile = '/Users/mtempleton/VOR_10.1063_5.0052678.xml'
+infile = '/proj/ads/abstracts/data/AIP/JATS.0609.new/RSI/v92/i6/064704_1/Markup/VOR_10.1063_5.0044438.xml'
```

Comment:
Shouldn't this be a local file?

Reply:
I'm considering pulling the examples out of pyingest for two reasons:
- How they get implemented depends on the overall workflow, which I think is going to be separated from pyingest itself eventually
- If we provide examples, we need to go back and select only those that are OA where the publisher wouldn't mind having their xml files public, because this is a public repo.